unity-client/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarVisibility.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using UnityEngine;
+using UnityEngine.Assertions;
+
+namespace DCL
+{
+    public class AvatarVisibility : MonoBehaviour
+    {
+
+        public AvatarRenderer renderer;
+        [Tooltip("Game objects that should be hidden/shown together with the renderer")]
+        public GameObject[] gameObjectsToToggle;
+        // When this property is true, then the visibility can't be modified
+        private Boolean locked = false;
```

Comment:
Why we are using `System.Boolean` here instead of just `bool`?

Reply:
Changed to `bool`